auth/user_mgt_test.go

```diff
@@ -576,32 +595,35 @@ func TestInvalidDeleteUser(t *testing.T) {
 }
 
 func TestMakeExportedUser(t *testing.T) {
-	rur := responseUserRecord{
-		UID:           "testuser",
-		Email:         "testuser@example.com",
-		PhoneNumber:   "+1234567890",
-		EmailVerified: true,
-		DisplayName:   "Test User",
-		ProviderUserInfo: []*UserInfo{
+
+	rur := &identitytoolkit.UserInfo{
+		LocalId:          "testuser",
+		Email:            "testuser@example.com",
+		PhoneNumber:      "+1234567890",
+		EmailVerified:    true,
+		DisplayName:      "Test User",
+		Salt:             "salt",
+		PhotoUrl:         "http://www.example.com/testuser/photo.png",
+		PasswordHash:     "passwordhash",
+		ValidSince:       1494364393,
+		Disabled:         false,
+		CreatedAt:        1234567890,
+		LastLoginAt:      1233211232,
+		CustomAttributes: `{"admin": true, "package": "gold"}`,
+		ProviderUserInfo: []*identitytoolkit.UserInfoProviderUserInfo{
 			{
-				ProviderID:  "password",
+				ProviderId:  "password",
 				DisplayName: "Test User",
-				PhotoURL:    "http://www.example.com/testuser/photo.png",
+				PhotoUrl:    "http://www.example.com/testuser/photo.png",
 				Email:       "testuser@example.com",
+				RawId:       "testuser@example.com",
```

Comment:
Ah I see why you added the LocalId property above. I'd recommend changing this to `testuid` so the intention is clearer.

Reply:
ok fix